### ISSUE TITLE
Include WBbt & FBbt anatomy ontologies

### ISSUE THREAD

**Issue by kevinschaper** [Knowledge-Graph-Hub/kg-phenio#64]
This will help to connect 400k & 100k gene expression edges respectively into MonarchKG.

https://obofoundry.org/ontology/fbbt.html
https://obofoundry.org/ontology/wbbt.html

**Comment by matentzn** [comment]
This issue is better placed in [phenio](https://github.com/monarch-initiative/phenio/issues), not KG phenio. Its odd though because both are in there:

https://github.com/monarch-initiative/phenio/blob/main/src/ontology/phenio-odk.yaml#L91
https://github.com/monarch-initiative/phenio/blob/main/src/ontology/phenio-odk.yaml#L93

**Comment by matentzn** [comment]
YOu dont see them in KG phenio?

**Comment by caufieldjh** [comment]
I also don't see them among the KG-Phenio nodes - just AnnotationProperty types like `OBO:fbbt#part_of` and `OBO:wbbt#part_of`

**Comment by matentzn** [comment]
Yes it is _very_ very important that we all use the same prefix map in monarch 

@kevinschaper I would suggest you use https://github.com/linkml/prefixmaps in all Monarch related projects!

**Comment by kevinschaper** [comment]
I think @caufieldjh is saying that they're OBO:FBbt_ in the Phenio graph?  The edges I'm dropping right now due to missing nodes have `FBbt:` and `WBbt:` prefixes - but I do see the OBO:FBbt_ and OBO:WBbt_ nodes!  Is that something you'll fix in KG Phenio? or should I fix them in our import preprocessing of KG Phenio?

I'll keep an eye out for were we can use prefixmaps. The case we have most often is that we're ingesting files that don't have an ID prefix, or have an ID prefix that we need to change - would it work to use prefixmaps almost like an enumeration to add guard rails?

**Comment by matentzn** [comment]
In this case it's probably KG phenio that needs to adapt the prefixmap library:)
